BaitDisplay.java

```diff
@@ -1,67 +1,114 @@
+    Cursor conditions_cursor;
+    String input_temp, input_clarity, input_structures, input_clouds, input_wind;
+    String nameText, descriptionText;
+    TextView name, description;
+    ImageView picView;
+    int picture;
+
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_bait_display);
-        FishingPlan inputs = new FishingPlan();
-        String input_temp = inputs.getWaterTempInput();
-        String input_clarity = inputs.getWaterClarityInput();
-        String input_structures = inputs.getStructureInput();
-        String input_clouds = inputs.getCloudInput();
-        String input_wind = inputs.getWindInput();
-        String[] input_array = {input_temp, input_clarity, input_clouds, input_structures, input_wind};
 
+
+        //Get the inputs from the Fishing Plan activity
+        input_temp = FishingPlan.water_temp_input;
+        input_clarity = FishingPlan.water_clarity_input;
+        input_structures = FishingPlan.structure_input;
+        input_clouds = FishingPlan.cloud_input;
+        input_wind = FishingPlan.wind_input;
+        //String[] input_array = {input_temp, input_clarity, input_clouds, input_structures, input_wind};
+        Log.i("input****", input_temp);
+        //Log.d("input****",input_temp);
         try{
-            SQLiteOpenHelper fishingPlanDatabaseHelper = new FishingTextbookDatabaseHelper(this);
-            db = fishingPlanDatabaseHelper.getReadableDatabase();
+            //Get an instance of the database : NewDatabaseThing. Then make it a writable database
+            SQLiteOpenHelper FishingPlanDatabaseHelper = new NewDatabaseThing(this);
+            Log.i("name**","here");
+            SQLiteDatabase db = FishingPlanDatabaseHelper.getWritableDatabase();
             //Create the cursor
-            Lure_conditions_cursor = db.query("LURE CONDITIONS", new String[] {"NAME", "CONDITIONS"},
-                        "_id = ?",new String[] {input_temp, input_clarity, input_clouds,
-                            input_structures, input_wind}
-                            ,null,null,null);
-            if (Lure_conditions_cursor.moveToFirst()) {
-                //Get drink details from cursor
-                String nameText = Lure_conditions_cursor.getString(0);
-                String descriptionText = Lure_conditions_cursor.getString(1);
-                int photoId = Lure_conditions_cursor.getInt(2);
-
-                //Populate the drink name
-//                TextView name = (TextView) findViewById(R.id.name);
-//                name.setText(nameText);
+            //First set of info
+            Cursor cursor = db.query("LURES", new String[] {"NAME","DESCRIPTION","IMAGE_RESOURCE_ID"},
+                    "NAME = 'Jig'",null,null,null,null);
+            accessCursor(cursor);
+            //Populate the Display fields
+            name = (TextView) findViewById(R.id.first_name);
```

Comment:
You could use [Android Data Binding Library](https://developer.android.com/topic/libraries/data-binding/index.html) to generate binding class links the layout variables with the Views within the layout